mqtt-gpio-monitor.ini.example

```diff
@@ -3,9 +3,12 @@ MODULE             = <gpio or pfio>
 DEBUG              = True
 
 MQTT_HOST          = localhost
-MQTT_PORT          = 1883
+MQTT_PORT          = 8883
 MQTT_USERNAME      = 
 MQTT_PASSWORD      = 
+MQTT_TLS_PROTOCOL  = tlsv1_2
```

Comment:
I think leave these blank by default (and leave port as `1883`) and have a commented section with your TLS defaults